pages/index.tsx

```diff
@@ -0,0 +1,584 @@
+                        }
+                      },
+                      {
+                        type_id: containTypeLinkId,
+                        to: {
+                          data: {
+                            type_id: formatTypeLinkId,
+                            string: { data: { value: sound.record["mimeType"] === "audio/webm;codecs=opus" ? "webm" : "aac" } },
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            {
+              type_id: containTypeLinkId,
+              to: {
+                data: {
+                  type_id: durationTypeLinkId,
+                  number: { data: { value: sound.record["msDuration"] } },
+                }
+              }
+            },
+            {
+              type_id: containTypeLinkId,
+              to: {
+                data: {
+                  type_id: startTimeTypeLinkId,
+                  string: { data: { value: sound.startTime } },
+                }
+              }
+            },
+            {
+              type_id: containTypeLinkId,
+              to: {
+                data: {
+                  type_id: endTimeTypeLinkId,
+                  string: { data: { value: sound.endTime } },
+                }
+              }
+            }]
+        }
+      })));
+      console.log("flakeed1");
+      console.log("soundLinkId", soundLinkId)
+
+      const { data: [isGcloudAuthKeyLinkId] } = await deep.select({
+        type_id: gcloudAuthKeyTypeLink,
+        in: {
+          type_id: containTypeLinkId,
+          from_id: deep.linkId,
+        },
+      });
+
+      console.log("googleauth", googleAuth);
+      if (!isGcloudAuthKeyLinkId) {
+        const parsedGoogleAuth = JSON.parse(googleAuth);
+        await deep.insert({
+          type_id: gcloudAuthKeyTypeLink,
+          object: { data: { value: parsedGoogleAuth } },
+          in: {
+            data: [
+              {
+                type_id: containTypeLinkId,
+                from_id: deep.linkId,
+              }
+            ]
+          }
+        })
+      }
+      console.log("flakeed3");
+
+      const { data: [{ id: transcribeTextLinkId }] } = await deep.insert({
+        type_id: transcribeTypeLinkId,
+        from_id: deep.linkId,
+        to_id: soundLinkId,
+        in: {
+          data: {
+            type_id: containTypeLinkId,
+            from_id: deep.linkId,
+          }
+        }
+      });
+      console.log("transcribeTextLinkId", transcribeTextLinkId)
+
+      await delay(8000);
+      const { data: [transcribedTextLinkId] } = await deep.select({
+        type_id: transcriptionTypeLinkId,
+        in: {
+          type_id: containTypeLinkId,
+          from_id: soundLinkId
+        },
+      });
+      console.log("transcribedTextLinkId", transcribedTextLinkId)
+
+      const { data: [isApiKeyLinkId] } = await deep.select({
```

Comment:
Why is this variable called isApiKeyLinkId instead of apiKeyLinkId? Why is there prefix 'is' but it is not a boolean value.
There is the same mistakes as before. This is required data and without it we should not show main content